src/table/components/TableWrapper.jsx

```diff
@@ -152,13 +154,23 @@ export default function TableWrapper(props) {
         data-testid="table-container"
       >
         <Table stickyHeader aria-label={tableAriaLabel}>
-          <TableHeadWrapper {...props} />
+          <TableHead>
+            <TableHeadWrapper
+              {...props}
+              setFocusedCellCoord={setFocusedCellCoord}
+              focusedCellCoord={focusedCellCoord}
+            />
+            {totalsPosition === 'top' && <TableTotals {...props} setFocusedCellCoord={setFocusedCellCoord} />}
+          </TableHead>
```

Comment:
Sorry I don't like this, having the `TableHead` component wrap the `TableHeadWrapper` doesn't make sense. I'd say move  `TableHead` back to the other file

Reply:
Let me check out your branch and see if my approach works

Reply:
It works, I have a commit I can push if you want. Sorry if I didn't explain it well enough. Yes, there is a reason. `TableHeader` adds no information to `TableTotals` component. Everything it needs exists inside `TableWrapper`. So, to reduce prop drilling (we want to keep it at the `TableWrapper` level, not lower) and keep the show logic in TableWrapper, for both top and bottom.

This is a bit long, but explains prop drilling and ways to solve it. What we are doing here is Container components, the second way described in the article

Reply:
yes and then you add children (which is on the props now) at the end of the `TableHeadWrapper` component here
```
      </TableRow>
      {children}
    </TableHead>

Reply:
@Max now I'm confused... Where exactly you want this component to be? inside `TableHeadWrapper.jsx` or inside `TableWrapper.jsx` ?
I know it works if I set it at the end of `TableHeadWrapper` as a child, it's where I had put it from the very beginning but then you said it is preferred to be set inside `TableWrapper.jsx` instead or I got it totally wrong?